site/tsconfig.json

```diff
@@ -1,3 +1,8 @@
 {
-    "include" : ["./ts/**/*.ts"]
+    "include" : ["./ts/**/*.ts", "./public/js/**/*.js"],
+    "compilerOptions": {
+        "strict": true,
+        "allowJs": true,
+        "noEmit": true
```

Comment:
You can remove the `-noEmit` from the "package.json" `build` script since its being added here now